lib/floe/workflow/runner/docker.rb

```diff
@@ -91,8 +148,35 @@ def run_container_params(image, env, secrets_file)
           params << image
         end
 
+        def wait_params(until_timestamp)
+          params = ["events", [:format, "{{json .}}"], [:filter, "type=container"], [:since, Time.now.utc.to_i]]
+          params << [:until, until_timestamp.to_i] if until_timestamp
+          params
+        end
+
+        def parse_notice(notice)
+          notice = JSON.parse(notice)
```

Comment:
Do we have any handling for if this doesn't parse? (I realize that's unlikely but I could see things where a partial unbuffered write is terminated mid-write and you don't get a full JSON line.